DEVELOPMENT.md

```diff
@@ -0,0 +1,64 @@
+
+## One Time Setup
+
+Install goose for db migrations:
+
+```
+$ go install github.com/pressly/goose/v3/cmd/goose@latest
+```
+
+## Create and Populate PostgreSQL Database
+
+Launch postgresql: 
+
+```bash
+podman run --name sippy-postgres -e POSTGRES_PASSWORD=password -p 5432:5432 -d quay.io/enterprisedb/postgresql
+```
+
+TODO: podman run postgres as a regular user may require disabling selinux on Fedora 35. Needs investigation.
+
+## Populating Data
+
+### From Scratch
+
+Populate db schema with goose, may need to run periodically if new migrations come in, fetch data from testgrid, and load it into the db. This process can take some time for recent OpenShift releases with substantial number of jobs. Using older releases like 4.7 can be fetched and loaded in just a couple minutes.
+
+```bash
+goose --dir dbmigration postgres "user=postgres password=password dbname=postgres sslmode=disable" up
+go build -mod=vendor . && ./sippy --local-data /opt/sippy-testdata --release 4.11 --fetch-data /opt/sippy-testdata --log-level=debug
+go build -mod=vendor . && ./sippy --release 4.11 --local-data /opt/sippy-testdata --load-database --log-level=debug --database-dsn="postgresql://postgres:password@localhost:5432/postgres" --skip-bug-lookup
```

Comment:
for line 28-29, you mean that you run line 27 and then you run one of 28 or 29 right?

Reply:
No you actually need all of the above. --fetch-data pulls testgrid files down to disk, and --load-database populates the db using those files.